### DISCUSSION TITLE
Add each custom prompt directly to the command palette

### DISCUSSION THREAD

**Discussion by kiradzS** [logancyang/obsidian-copilot#1172] [Q&A]
Could you please add an option to run each custom prompt directly from the command palette? This would make it much easier to use frequently used prompts. Thank you!

**Comment by logancyang** [comment]
You can assign shortcut to "apply custom prompt" or use slash command in chat input.

Adding custom prompts directly to the command palette is technically possible I guess, but it can overwhelm the command palette pretty quickly, so I chose a 2-level command instead.

If anyone has a solution assigning shortcut directly to individual custom prompt, let me know. In the meantime please comment on this issue if you like or dislike the idea of flattening the custom prompt commands directly into the command palette.

**Comment by kiradzS** [comment]
I got it! I noticed that in the QuickAdd plugin, custom commands have a small lightning button that controls whether they appear in the command palette. If it's not too difficult to implement, would it be possible to consider this for a future update🤩
